requirements.txt

```diff
@@ -1,6 +1,6 @@
-genshi
-lxml
-pandas>=1.2
-pyyaml
-requests
-openpyxl
+genshi==0.7.7
```

Comment:
I agree it is not ideal for people that do not make use of environments, but >= on pandas would not have prevented the issue we now have for example, since certain things will get deprecated. <= would that be better than?

Reply:
Both we need bigger than 2.0 and smaller like 3.0 left?